iati/schemas.py

```diff
@@ -96,7 +124,7 @@ def _change_include_to_xinclude(self, tree):
         include_xpath = (iati.constants.NAMESPACE + 'include')
         include_el = tree.getroot().find(include_xpath)
         if include_el is None:
-            return
+            return tree
```

Comment:
Is there a reason why this necessary here? Are we just returning something so that it

just asking as I have an ulterior motive, given my work in the `get-data-from-schema` branch and I'm trying to prevent more merge conflicts and more work to get that one ready soon (which admittedly isn't a problem for this PR)!

Reply:
Bug fix so that it returns a tree rather than `None`.

This doesn't seem to be in the changelog.

Reply:
019816be75d4a2cd89b04489fcb809f646341564